plugins/package-configuration-providers/dir/src/main/kotlin/DirPackageConfigurationProvider.kt

```diff
@@ -36,46 +39,46 @@ data class DirPackageConfigurationProviderConfig(
     /**
      * The path of the package configuration directory.
      */
-    val path: File,
+    val path: String,
```

Comment:
Maybe it's worth mentioning in the commit message that this needs to be changed to `String` as `File` is not a supported plugin option type?